packages/server/prisma/schema.prisma

```diff
@@ -11,6 +11,7 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+// Test models
```

Comment:
We should remove test models after this and start testing api/queries with "Real" models. This should be done in another PR where we also remove those deprecated lambdas and endpoints.

Reply:
Yup. Once we have tested this as working in main, we can start removing those.